src/plugins/vis_type_timeline/server/index.ts

```diff
@@ -61,6 +61,8 @@ export const config: PluginConfigDescriptor<ConfigSchema> = {
     renameFromRoot('timelion.ui.enabled', 'vis_type_timeline.ui.enabled', true),
     renameFromRoot('vis_type_timelion.ui.enabled', 'vis_type_timeline.ui.enabled', true),
     renameFromRoot('timeline.ui.enabled', 'vis_type_timeline.ui.enabled', true),
+
+    renameFromRootWithoutMap('graphiteBlockedIPs', 'graphiteDeniedIPs'),
```

Comment:
should be `renameFromRootWithoutMap('vis_type_timeline.graphiteBlockedIPs', 'vis_type_timeline.graphiteDeniedIPs')`,